srcpkgs/pass-import/template

```diff
@@ -1,20 +1,22 @@
 # Template file for 'pass-import'
 pkgname=pass-import
-version=3.1
+version=3.2
 revision=1
-build_style=gnu-makefile
-hostmakedepends="python3-setuptools python3-yaml"
-depends="pass>=1.7.0 python3-defusedxml python3-magic
- python3-cryptography python3-SecretStorage python3-pykeepass"
-checkdepends="pass python3-defusedxml python3-magic python3-cryptography
- python3-SecretStorage python3-pykeepass"
+build_style=python3-module
+hostmakedepends="python3-setuptools"
+depends="pass python3-defusedxml python3-magic python3-cryptography python3-SecretStorage python3-pykeepass python3-yaml"
+checkdepends="python3-pytest $depends"
 short_desc="Pass extension for importing data from most existing password managers"
 maintainer="Alan Brown <adbrown@rocketmail.com>"
 license="GPL-3.0-or-later"
 homepage="https://github.com/roddhjav/pass-import"
 distfiles="https://github.com/roddhjav/pass-import/releases/download/v${version}/pass-import-${version}.tar.gz"
-checksum=659d2b503a18d57c9d63dee216d042d0e24f88576cdd0c7fc4b3426e40dbdbc2
+checksum=d943df14dcfec063a6c881c315ce65813db6e105b14a9b0156a73dcf7e529b76
 
 do_check() {
-	python3 setup.py test
+	# from flake8
+	PYTHONPATH=build/lib python3 -m pytest \
+		--deselect=tests/exports/test_keepass.py \
+		--deselect=tests/imports/test_parse.py
+
 }
```

Comment:
you don't need to include the from flake8 comment. Would adding deselects to make_check_args and dropping do_check work?